python/pyarrow/tests/test_cuda.py

```diff
@@ -119,6 +119,24 @@ def make_random_buffer(size, target='host'):
     raise ValueError('invalid target value')
 
 
+def test_copy_from_buffer():
+    arr, buf = make_random_buffer(128)
+    cudabuf = global_context.buffer_from_data(buf)
+
+    mm2 = global_context1.memory_manager
+    for dest in [mm2, mm2.device]:
+        buf2 = cudabuf.copy(dest)
+        assert buf2.device_type == pa.DeviceAllocationType.CUDA
```

Comment:
Perhaps also try copying directly to the CPU?
```suggestion
        assert buf2.device_type == pa.DeviceAllocationType.CUDA
        assert buf2.copy(pa.default_memory_manager()).equals(buf)
```

Reply:
Ah, so the `pa.default_memory_manager()` is the memory manager for CPU.

Reply:
It is `default_cpu_memory_manager`* =)

Reply:
AH, sorry :-)